### ISSUE TITLE
AttributeError: 'PosixPath' object has no attribute 'write'. Did you mean: 'drive'?

### ISSUE THREAD

**Issue by marcbrevoort-cyberhive** [pypa/pip-audit#439]
Thank you for reporting a potential bug in `pip-audit`! Please read the next parts of this template carefully:

**IMPORTANT**: Please **do not** report auditing errors (false positives or negatives) to this repository. Instead, please report them to [pypa/advisory-database](https://github.com/pypa/advisory-database/issues/new).

**IMPORTANT:** Please fill out every section below. Bug reports with missing information will be
given a lower priority or closed outright.

Please comment out or remove this line and everything above it from your report.

## Bug description

As of 15 December nightly (i.e. pip-audit released 14 December), it reports
```
No known vulnerabilities found
Traceback (most recent call last):
  File "/opt/venv/bin/pip-audit", line 8, in <module>
    sys.exit(audit())
  File "/opt/venv/lib/python3.10/site-packages/pip_audit/_cli.py", line 543, in audit
    print(formatter.format(result, fixes), file=args.output)
AttributeError: 'PosixPath' object has no attribute 'write'. Did you mean: 'drive'?
make: *** [Makefile:148: safety] Error 1
```

## Reproduction steps

triggered by running
```
pip3 install pip-audit && pip3 freeze > /tmp/audit.txt && pip-audit --no-deps --requirement /tmp/audit.txt --ignore-vuln PYSEC-2020-220 --ignore-vuln PYSEC-2021-125 --ignore-vuln PYSEC-2020-221
```
Workaround: Pinning to version 2.4.5 instead:
```
pip3 install pip-audit==2.4.5 && pip3 freeze > /tmp/audit.txt && pip-audit --no-deps --requirement /tmp/audit.txt --ignore-vuln PYSEC-2020-220 --ignore-vuln PYSEC-2021-125 --ignore-vuln PYSEC-2020-221
```

## Expected behavior

No error after finding no vulnerabilities.

## Screenshots and logs

DEBUG:pip_audit._cli:parsed arguments: Namespace(cache_dir=None, desc=<VulnerabilityDescriptionChoice.Auto: 'auto'>, dry_run=False, extra_index_urls=[], fix=False, format=<OutputFormatChoice.Columns: 'columns'>, ignore_vulns=['PYSEC-2020-220', 'PYSEC-2021-125', 'PYSEC-2020-221'], index_url='https://pypi.org/simple/', local=False, no_deps=True, output=PosixPath('stdout'), paths=[], progress_spinner=<ProgressSpinnerChoice.On: 'on'>, project_path=None, require_hashes=False, requirements=[<_io.TextIOWrapper name='/tmp/audit.txt' mode='r' encoding='UTF-8'>], skip_editable=False, strict=False, timeout=15, verbose=True, vulnerability_service=<VulnerabilityServiceChoice.Pypi: 'pypi'>)
WARNING:pip_audit._cli:--no-deps is supported, but users are encouraged to fully hash their pinned dependencies
WARNING:pip_audit._cli:Consider using a tool like `pip-compile`: https://pip-tools.readthedocs.io/en/latest/#using-hashes
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/ansible/6.7.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/ansible-core/2.13.7/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/asgiref/3.5.2/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/backports-zoneinfo/0.2.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/black/22.12.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/cachecontrol/0.12.11/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/certifi/2022.12.7/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/cffi/1.15.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 115
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 115
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/cfgv/3.3.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/charset-normalizer/2.1.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/click/8.1.3/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/commonmark/0.9.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/conttest/0.0.8/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/cryptography/38.0.4/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/cyclonedx-python-lib/3.1.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/distlib/0.3.6/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/django/4.1.4/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/environs/9.5.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/filelock/3.8.2/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/flake8/6.0.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/html5lib/1.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/identify/2.5.9/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/idna/3.4/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/isort/5.11.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/jinja2/3.1.2/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/lockfile/0.12.2/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/markupsafe/2.1.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/marshmallow/3.19.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/mccabe/0.7.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/model-bakery/1.9.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/msgpack/1.0.4/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/mypy-extensions/0.4.3/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/nodeenv/1.7.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/packageurl-python/0.10.4/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/packaging/21.3/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pathspec/0.10.3/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pip-api/0.0.30/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pip-requirements-parser/32.0.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pip-audit/2.4.9/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pkg-resources/0.0.0/json" in the cache
DEBUG:cachecontrol.controller:No cache entry available
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): pypi.org:443
DEBUG:urllib3.connectionpool:https://pypi.org:443 "GET /pypi/pkg-resources/0.0.0/json HTTP/1.1" 404 24
DEBUG:cachecontrol.controller:Status code 404 not in (200, 203, 300, 301, 308)
DEBUG:pip_audit._service.pypi:Dependency not found on PyPI and could not be audited: pkg-resources (0.0.0)
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/platformdirs/2.6.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pre-commit/2.20.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pycodestyle/2.10.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pycparser/2.21/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pyflakes/3.0.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pygments/2.13.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pyparsing/3.0.9/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/python-dotenv/0.21.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/pyyaml/6.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/rebl/1.1.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/requests/2.28.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/resolvelib/0.8.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/rich/12.6.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/semantic-version/2.10.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/setuptools-rust/1.5.2/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/six/1.16.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/sortedcontainers/2.4.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/sqlparse/0.4.3/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/toml/0.10.2/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/tomli/2.0.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/typing-extensions/4.4.0/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/urllib3/1.26.13/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/virtualenv/20.17.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
DEBUG:cachecontrol.controller:Looking up "https://pypi.org/pypi/webencodings/0.5.1/json" in the cache
DEBUG:cachecontrol.controller:Current age based on date: 114
DEBUG:cachecontrol.controller:Freshness lifetime from max-age: 900
DEBUG:cachecontrol.controller:The response is "fresh", returning cached response
DEBUG:cachecontrol.controller:900 > 114
No known vulnerabilities found
Traceback (most recent call last):
  File "/home/mrjb/git/cyberhive-meshvpn/venv/bin/pip-audit", line 8, in <module>
    sys.exit(audit())
  File "/home/mrjb/git/cyberhive-meshvpn/venv/lib/python3.8/site-packages/pip_audit/_cli.py", line 543, in audit
    print(formatter.format(result, fixes), file=args.output)
AttributeError: 'PosixPath' object has no attribute 'write'
 

## Platform information

ubuntu 20.04
pip-audit 2.4.9
python 3.8.10
pip 22.2.2

## Additional context

Running on Jenkins nightly, also reproducible on dev box.

**Comment by tetsuo-cpp** [comment]
Hi @marcbrevoort-cyberhive. Thanks for reporting this!

A fix has been merged in #437. We'll have a new release shortly.

**Comment by woodruffw** [comment]
2.4.10 has been released with the fix. Thanks again for reporting @marcbrevoort-cyberhive!
